### PR TITLE
reduce config items

### PR DESCRIPTION
**Pull request by lucklove**
Signed-off-by: lucklove <gnu.crazier@gmail.com>

### CODE REVIEW

**Comment by zhouqiang-cl** [comment]
@AndreMouche @zhangjinpeng1987 @nolouch @jackysp @zz-jason @winkyao @ericsyh PTAL

File: conf/tidb.yml [hunk 1]
```diff
@@ -106,39 +61,10 @@ performance:
...
-
-  # Probability to use the query feedback to update stats, 0 or 1 for always false/true.
-  # feedback-probability: 0.05
-
-  # The max number of query feedback that cache in memory.
-  # query-feedback-limit: 1024
-
-  # Pseudo stats will be used if the ratio between the modify count and
-  # row count in statistics of a table is greater than it.
-  # pseudo-estimate-ratio: 0.8
```
Comment (zz-jason) [inline conf/tidb.yml]: It's better to keep them in the config files:
```
run-auto-analyze
feedback-probability
query-feedback-limit
pseudo-estimate-ratio
```

Comment (lucklove) [reply conf/tidb.yml]: https://docs.google.com/spreadsheets/d/1fmY3MSklHb9jYQOv3Djcrhj3Uz97fI06soaMy4OKcRs/edit#gid=0

PM think they should not be visible to user

File: conf/tidb.yml [hunk 2]
```diff
@@ -44,57 +31,25 @@ log:
...
-  # Disable automatic timestamps in output
-  # disable-timestamp: false
-
   # Queries with execution time greater than this value will be logged. (Milliseconds)
   # slow-threshold: 300
 
   # Queries with internal result greater than this value will be logged.
   # expensive-threshold: 10000
 
-  # Maximum query length recorded in log.
```
Comment (winkyao) [inline conf/tidb.yml]: we need to mention these removed config items in our [documents](https://pingcap.com/docs-cn/v3.0/reference/configuration/tidb-server/configuration-file/).

Comment (lucklove) [reply conf/tidb.yml]: OK, I'll check it

File: conf/tikv.yml [hunk 3]
```diff
@@ -364,71 +272,13 @@ rocksdb:
...
-  # info-log-keep-log-file-num: 10
-
-  ## Specifies the RocksDB info log directory.
-  ## If it is empty, the log files will be in the same directory as data.
-  ## If it is not empty, the log files will be in the specified directory, and the DB data directory's
-  ## absolute path will be used as the log file name's prefix.
-  # info-log-dir: ""
-
-  ## Options for "Titan"
-  titan:
```
Comment (Connor1996) [inline conf/tikv.yml]: why remove this? 🤣can you add it back?

Comment (liubo0127) [reply conf/tikv.yml]: All config items which been reduced have been added back.

Comment (Connor1996) [reply conf/tikv.yml]: which release version

Comment (liubo0127) [reply conf/tikv.yml]: master、3.1 and 3.0

Comment (Connor1996) [reply conf/tikv.yml]: didn't see it in v3.0.7 and master

Comment (liubo0127) [reply conf/tikv.yml]: Update, added back in 3.0 and 3.1, master did not add. The released version can't be added back.

Comment (yiwu-arbug) [reply conf/tikv.yml]: Can we also revert the change for titan in master?

**Pull request merged**